propagation_test.go

```diff
@@ -0,0 +1,57 @@
+package opentracing
+
+import (
+	"net/http"
+	"testing"
+)
+
+const testHeaderPrefix = "testprefix-"
+
+func TestHTTPHeaderInject(t *testing.T) {
```

Comment:
Meh... I see what you're saying but I'd like to leave this as-is since – if nothing else – it makes it easy to quickly see the implications for various API changes. If you feel strongly I can make this change, though.